app/models/action_token.rb

```diff
@@ -7,7 +7,7 @@
 # A password reset key is valid for a limited time and may safely be left unused.
```

Comment:
Not actually for this PR changes, but initial comments for action tokens could be more general, those are propably left from old model.